app/scripts/background.js

```diff
@@ -96,7 +96,7 @@ const initApp = async (remotePort) => {
   log.info('MetaMask initialization complete.');
 };
 
-if (isManifestV3()) {
+if (isManifestV3) {
```

Comment:
It'd be nice to have a separate PR that turns it into a boolean, adds a unit test ensuring it's not turned back into a function and replace all usage, all in one step. So that when merged, there's no chance of a hard to find bug.

Reply:
Or at least add the test :)

Reply:
I removed the change I will create a separate PR for it.